.github/workflows/compliance.yml

```diff
@@ -13,3 +13,14 @@ permissions:
 jobs:
   compliance:
     uses: open-sauced/hot/.github/workflows/compliance.yml@main
+name: reviewdog
```

Comment:
This already has a name on line 1, did you mean to edit it?

Reply:
No @nickmccurdy.

Reply:
```suggestion
```